img/atom.svg

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
can you make sure all of these are the same height? in the attached screenshot they look slightly uneven